src/Components/WebView/WebView/src/Microsoft.AspNetCore.Components.WebView.csproj

```diff
@@ -15,6 +15,10 @@
   <ItemGroup>
     <InternalsVisibleTo Include="Microsoft.AspNetCore.Components.WebView.Test" />
   </ItemGroup>
+  
+  <ItemGroup>
+    <EmbeddedResource Update="Resources.resx" ExcludeFromManifest="true" />
```

Comment:
Why would it be included at all to begin with?

Reply:
Because the embedded manifest file provider by default includes all embedded resources in it (don't ask me why we chose that default).

Reply:
OK that's definitely weird, but good to know!